src/form/bf_mx_lin.jl

```diff
@@ -172,3 +172,101 @@ function constraint_mc_storage_losses(pm::AbstractUBFModels, i::Int; nw::Int=nw_
         qsc + q_loss
     )
 end
+
+## KIG modifications
```

Comment:
Should have a docstring of the format:

```julia
@doc raw"""
    constraint_mc_load_power(pm::LPUBFDiagModel, load_id::Int; nw::Int=nw_id_default, report::Bool=true)

Short description here

latex math equations here
"""
```

See https://github.com/lanl-ansi/PowerModelsONM.jl/blob/main/src/form/shared.jl#L1-L12 as an example

the `@doc raw"""` part is necessary for inserting math equations due to the usage of `\`, so that Julia doesn't throw an error about invalid escape sequences